packages/ui/src/shared/constants/index.ts

```diff
@@ -3,6 +3,14 @@ export const YT_CYPRESS_COOKIE_NAME = 'YTCypressCookie';
 export const YT_LOCAL_CLUSTER_ID = 'ui';
 export const ODIN_PAGE_ID = 'odin';
 
+export const YT_OAUTH_ACCESS_TOKEN_NAME = 'yt_oauth_access_token';
+export const YT_OAUTH_REFRESH_TOKEN_NAME = 'yt_oauth_refresh_token';
+
+export enum AuthWay {
+    OAuth = 'oauth',
+    YT = 'yt',
+}
```

Comment:
It is always better to use `const enum` or union of strings type instead of regular enums  when it is possible, cause it allows to avoid unnecessary links between modules. `const enum`-s and union of strings [generate no code after compilation](https://www.typescriptlang.org/play?#code/KYOwrgtgBAggQgYSgbwFBQ1AhlAvFAciwIBp1MAjPQi08jAY2oIYNQF9VUGB7EAZwAuUUJCgARAKIAxFPSgATZgrqYRzYKswAzZtradUggJ4AHYFADiACQCSzAOYEoAH0IALZ24IBLAgG4uXgFhLAAuWERqeAQAOixA4KFFCKlZfDTYhUS+ZIdHANQgA). 

Also it is better to use more specific value instead of `'yt'` like `'yt-passwd'`/`'yt-pwd'` it allows easier to find places of usage in the code like [authWay === 'yt'](https://github.com/ytsaurus/ytsaurus-ui/pull/217/files#diff-60df5594b756d491317c43eca0b4e3e368e0728b15b706a9c2328db977c288c1R164). 
By the way you use comparison with a literal string so union of strings is more natural:
```ts
type AuthWay = 'YT_OAUTH' | 'YT_PASSWD';
```